### ISSUE TITLE
Calendar multiple dates

### ISSUE THREAD

**Issue by nutdesign932** [wpeventmanager/wp-event-manager#1686]
Hi,

We have the calendar add on of the plugin installed on our website but have noticed an issue when we have a multiple day event e.g. a two day event that starts on Saturday and finishes on Sunday for example, it only shows on the calendar for the first day and we need it to show across all the days that the event is running if that's possible.

Many thanks in advance!

**Comment by ritakikani** [comment]
@nutdesign932 , yes this is possible and for that you need to override template file and I will guide you to achieve this functionality but first you need to give me your site/page URL where you want this change.

Thank you.
